src/java.base/share/classes/java/security/Permissions.java

```diff
@@ -579,9 +579,12 @@ public boolean implies(Permission permission) {
     /**
      * @serialData Default fields.
      */
```

Comment:
Another ignored comment. I suggest just changing these to `/*` comments.

Reply:
Good idea.  Done.